guides/shop-managers-guide/01-create-a-user.md

```diff
@@ -0,0 +1,9 @@
+# Create a user
+
+Congratulations on a successful installation of Open Commerce. Now you should have the admin portal available at http://localhost:4080/
+![Admin](_assets/01-admin-clean.png)
+To create a user click on the profile icon in the top right corner on the screen. It will show you options to Sign In or Create Account. 
+![Admin sign in/ sign up](_assets/01-admin-signin-create.png)
+Click on Create Account to bring up the sign up window. 
+![Admin sign up](_assets/01-admin-sign-up.png)
```

Comment:
I think we should have some line breaks to break up this text to make it a little easier to read